src/constants/denoms.ts

```diff
@@ -0,0 +1,9 @@
+import { isMainnet } from './networks';
+
+export const OSMO_USDC_IBC_DENOM = isMainnet
```

Comment:
[suggestion] i think right now we have a `tokens` property in the `v1/env` config where we define the dydx usdc ibc denom. would probably be worth adding into there in case we need it in other places later?